src/bindings/bnd_mesh.cpp

```diff
@@ -715,6 +715,28 @@ BND_TUPLE BND_MeshFaceList::GetFace(int i) const
   return rc;
 }
 
+BND_TUPLE BND_MeshFaceList::GetFaceVertices(int faceIndex) const
+{
+  ON_MeshFace& face = m_mesh->m_F[faceIndex];
+  BND_TUPLE rc = CreateTuple(5);
+  SetTuple<bool>(rc, 0, true);
```

Comment:
You shouldn't have to have <bool> here, but the syntax is still ok

Reply:
please clarify

Reply:
You don't understand my gibberish writing skills? :)

You don't need the `<bool>` on SetTuple. C++ can figure out that this template is for a bool, but it doesn't really matter. Both techniques compile to the same thing.